public/kibi_timeline.js

```diff
@@ -164,7 +164,8 @@ define(function (require) {
             var endRawFieldValue;
 
             _.each(searchResp.hits.hits, function (hit) {
-              if (hit.fields[params.startField] === null || hit.fields[params.startField] === undefined) {
+              if (hit.fields[params.startField] === null || hit.fields[params.startField] === undefined
```

Comment:
Why we need this check here ?
In line 178 we are already getting the startValue and there is elready **if** there
Why we can NOT just add else to it ??
And why we have to test hit.fields against all 3 different values like null, undefined and NaN
I think the  getDescendantPropValue should return null if there is no value ?
Please ad a test for this case (there are already tests for this method) and then if it is true please remove this unnecessary checks and just print warning in else block